.circleci/config.yml

```
@@ -75,11 +75,14 @@ references:
   deploy_cluster: &deploy_cluster
    run:
      name: Deploy the job on the kubernetes cluster
+     no_output_timeout: 100m
```

Addresses https://github.com/PyTorchLightning/pytorch-lightning/pull/11098#issuecomment-1078641221.

```suggestion
     no_output_timeout: 100m  # by default, the job terminates after 10min of no output.
```

https://support.circleci.com/hc/en-us/articles/360045268074--Too-long-with-no-output-exceeded-10m0s-context-deadline-exceeded